docs/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+# Issue
+
+Thanks for opening an issue on the Fuel Indexer project.
+
+_*If you're an internal FuelLabs contributor, please feel free to remove this template when creating issues, as this template is meant for external users and contributors_.
+
+## Prerequisites
+
+- [ ] Can you reproduce the problem?
+- [ ] Do you know which version of the indexer you're using?
+- [ ] Did you already check for an answer in previous [Discussions](https://github.com/FuelLabs/fuel-indexer/discussions)?
+
+### Description
+
+- Please describe your issue in detail.
+- The more accurate detailed information you give, the quicker we can help you.
```

Comment:
I'd change this to:
> Detailed information helps us narrow down any potential causes.

Mainly, I'm weary of any guarantee of speed, no matter how vague it may be.